src/core/agent/services/multiSigService.test.ts

```diff
@@ -504,7 +509,9 @@ describe("Creation of multi-sig", () => {
       })
     );
 
-    (memberMetadataRecord.groupMetadata as any).groupCreated = false;
+    if(memberMetadataRecord.groupMetadata)
+      memberMetadataRecord.groupMetadata.groupCreated = false;
```

Comment:
Can we do `memberMetadataRecord.groupMetadata!.groupCreated = false;` to avoid the if statement?

Reply:
When we look at the core library, I think we can decide if we should disable this rule or not globally. We use it in many places. Will revisit